### DISCUSSION TITLE
websocket: RSV1 set, RSV2 set, FIN not set on control

### DISCUSSION THREAD

**Discussion by SilvaRaulEnriqueCJM** [gorilla/.github#21] [Q&A]
In local websocket connection is no problem with localhost, but if remote connection the problem is:
Read from server failed: websocket: RSV1 set, RSV2 set, FIN not set on control

¿What is the solution?

**Comment by coreydaley** [comment]
Can you supply a reproducer and some of the error messages that you are getting?
